src/components/common/verify-code/verify-code-controller.ts

```diff
@@ -114,7 +117,10 @@ export function verifyCodePost(
           if (options.journeyType !== JOURNEY_TYPE.PASSWORD_RESET_MFA) {
             nextEvent = USER_JOURNEY_EVENTS.PASSWORD_RESET_INTERVENTION;
           }
-        } else if (accountInterventionsResponse.data.temporarilySuspended) {
+        } else if (
+          accountInterventionsResponse.data.temporarilySuspended &&
```

Comment:
Just a note that you don't need this line anymore since it's checked in the below function :-) 

I also wonder about moving the "isSuspendedWithoutUserActions" to somewhere shared so we don't have two duplicated functions. Happy to have a chat about where we think that shared function should go if you don't have any immediate instinct